### PR TITLE
Add BUFG PIP fuzzer

### PR DESCRIPTION
**Pull request by litghost**
This solves 2 bits from #684.

### CODE REVIEW

**Comment by mithro** [comment]
Can you add a `README.md`?

File: fuzzers/044-clk-bufg-pips/Makefile [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export FUZDIR=$(shell pwd)
```
Comment (mithro) [inline fuzzers/044-clk-bufg-pips/Makefile]: Normal way to do this is;
```make
mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
```

File: fuzzers/044-clk-bufg-pips/bits.dbf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+26_07 26_08 27_06,CLK_BUFG.CLK_BUFG_BUFGCTRL0_I0.CLK_BUFG_IMUX28_0
```
Comment (mithro) [inline fuzzers/044-clk-bufg-pips/bits.dbf]: Would be good to add some information about how this file was worked out / created? Maybe that will be in your README?

File: fuzzers/044-clk-bufg-pips/build_zdb.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+""" Tool for building ZDB for BUFG pips.
```
Comment (mithro) [inline fuzzers/044-clk-bufg-pips/build_zdb.py]: zdb?

File: fuzzers/044-clk-bufg-pips/bits.dbf (hunk 2 above)
Comment (litghost) [reply fuzzers/044-clk-bufg-pips/bits.dbf]: None of the other fuzzers have ever needed that?

File: fuzzers/044-clk-bufg-pips/Makefile (hunk 1 above)
Comment (litghost) [reply fuzzers/044-clk-bufg-pips/Makefile]: That might be the normal way, but it seems less readable to me.

**Comment by litghost** [comment]
Superceeded by #727.

**Pull request closed without merging**